extensions.json

```
@@ -160,6 +160,14 @@
       "author": "Marien Fressinaud",
       "url": "https://github.com/FreshRSS/Extensions",
       "type": "gh-subdirectory"
+    },
+    {
+      "name": "GReader Redate",
+      "description": "Workaround a dating problem on GReader API with Reeder and EasyRSS.",
```

```suggestion
      "description": "Works around a dating problem on GReader API with Reeder.",
```

Or "Workaround for".

And I'd indeed leave off the EasyRSS, or at least put in something like "prior to 1.7.0".